src/libutil/util.hh

```diff
@@ -172,10 +172,17 @@ constexpr char treeNull[] = "    ";
 
 
 /**
- * Base64 encoding/decoding.
+ * Encode arbitrary bytes as Base64.
  */
 std::string base64Encode(std::string_view s);
-std::string base64Decode(std::string_view s);
+
+/**
+ * Decode arbitrary bytes to Base64.
+ *
+ * @param sensitiveValue Avoid displaying the raw Base64 in error messages,
+ * e.g. to avoid leaking private keys.
+ */
+std::string base64Decode(std::string_view s, bool sensitiveValue = false);
```

Comment:
```suggestion
std::string base64Decode(std::string_view s, bool sensitiveValue);
```

Each caller should just make a decision.

Or, if we disagree, it should fail safely:

```suggestion
std::string base64Decode(std::string_view s, bool sensitiveValue = true);
```